osu.Game/Overlays/BeatmapSet/Info.cs

```diff
@@ -162,12 +185,12 @@ public MetadataSection(string title)
                 {
                     header = new OsuSpriteText
                     {
-                        Text = title,
+                        Text = this.type.ToString(),
```

Comment:
`this.` should not be used if unnecessary.

Reply:
Does CI allows this?